config.json.sample

```diff
@@ -1,6 +1,7 @@
 {
   "zk": "zk://127.0.0.1:2181/mesos",
   "masters": ["127.0.0.1:5050"],
+  "stateTimeoutSeconds": 300,
```

Comment:
One of the questions I have is why we didn't use strings, and time.Duration / time.ParseDuration? @jdef do you know?